subsys/usb/class/mass_storage.c

```diff
@@ -481,7 +482,7 @@ static void CBWDecode(u8_t *buf, u16_t size)
 				} else {
 					usb_ep_set_stall(
 					    CONFIG_MASS_STORAGE_IN_EP_ADDR);
-					SYS_LOG_DBG("BI-STALL");
+					SYS_LOG_WRN("BI-STALL");
```

Comment:
if this is going to be a warning, we need a proper warning message, what is the warning here?